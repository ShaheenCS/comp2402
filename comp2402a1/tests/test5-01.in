sleep
bicycle
adaptable
room
hunter
be
programmer
forest
abortion
cat
