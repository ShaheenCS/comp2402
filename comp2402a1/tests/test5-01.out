be
cat
room
sleep
forest
hunter
bicycle
abortion
adaptable
programmer
